cartridge/graphql/execute.lua

```diff
@@ -213,6 +213,10 @@ local function completeValue(fieldType, result, subSelections, context, opts)
   end
 
   if fieldTypeName == 'Object' then
+    if type(result) ~= 'table' then
```

Comment:
And `if not util.is_map(result)` here?

Reply:
It's redundant I think. Because you should declare nested fields for map. And array won't have them. In contrast map was interpreted as an empty array thus is_array check makes sense.

Reply:
I don't get it. Object queries with no selections are not allowed in graphql. So this check may be done. If at least one field is not nullable, then resolver has to return an object with at least one string key.

Anyway, on the other hand there is a particular case, when an object has all fields nullable. There will be empty map indeed, which is indistinguishable with an empty array in lua. So lets leave this peculiarity as is for now.